modules/nixos/home-assistant/config/configuration.yaml

```diff
@@ -0,0 +1,4 @@
+http:
+  use_x_forwarded_for: true
+  trusted_proxies:
+    - 127.0.0.1
```

Comment:
this file is not part of the build and hass only works right now because you manually copied it to the container